domain/media/src/test/kotlin/com/giraffe/media/series/usecase/topRated/ObserveTopRatedSeriesUseCaseTest.kt

```diff
@@ -0,0 +1,31 @@
+package com.giraffe.media.series.usecase.topRated
+
+import com.giraffe.media.collections.util.createFakeSeries
+import com.giraffe.media.series.repository.SeriesRepository
+import com.giraffe.media.util.limit
+import com.google.common.truth.Truth.assertThat
+import io.mockk.coEvery
+import io.mockk.mockk
+import kotlinx.coroutines.flow.flowOf
+import kotlinx.coroutines.test.runTest
+import org.junit.jupiter.api.Test
+
+class ObserveTopRatedSeriesUseCaseTest {
+    private val repository: SeriesRepository = mockk()
+    private val useCase: ObserveTopRatedSeriesUseCase = ObserveTopRatedSeriesUseCase(repository)
+
+
+    @Test
+    fun `given top rated series, when invoke is called, then return series list`() = runTest {
```

Comment:
```suggestion
    fun `should return list of top rated series when observeTopRated function in repository called'
```
I think this name is more clear